mu-plugins/blocks/multisite-latest-posts/src/block.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * WordPress dependencies
+ */
+import { Fragment, useEffect, useState } from '@wordpress/element';
+import { gmdate, gmdateI18n } from '@wordpress/date';
+import { __ } from '@wordpress/i18n';
+
+const getCategory = ( embed, id ) => {
+	if ( ! embed[ 'wp:term' ] ) {
+		return;
+	}
+
+	for ( let idx = 0; idx < embed[ 'wp:term' ].length; idx++ ) {
+		const terms = embed[ 'wp:term' ][ idx ];
+
+		return terms.find( ( term ) => term.id === id );
+	}
+};
+
+const Block = ( { endpoint, itemsToShow } ) => {
+	const [ posts, setPosts ] = useState( [] );
+	const [ error, setError ] = useState();
+	const [ loading, setLoading ] = useState( false );
+
+	const getPosts = async () => {
+		setLoading( true );
+		const response = await fetch( `${ endpoint }/posts?per_page=${ itemsToShow }&_embed=true` );
```

Comment:
Would `apiFetch` be useful here?

Reply:
I don't think so. Is there anything it provides that would help?

Reply:
Maybe browser compat? I'm not totally sure for this use case (it has some nice handling of the WP API, but I'm not sure how well that will translate to a separate WP install).

Reply:
I'll take a look.

Reply:
It looks like it does have some better error handling that could be useful.  

I couldn't get it to work using:
`apiFetch.use( apiFetch.createRootURLMiddleware( rootURL ) );`

Stepping through the code, it was updating the rootURL but doing so **after** triggering the ajax request, even when I created the middleware outside of the component itself. Not sure what was going on, but I think I'm comfortable not making use of it anyways.

I've added some better error handling which I think is all we need.